utxo/utxo.go

```diff
@@ -1889,6 +1889,23 @@ func (uv *UtxoVM) queryContractMethodACLWithConfirmed(contractName string, metho
 	return uv.aclMgr.GetContractMethodACLWithConfirmed(contractName, methodName)
 }
 
+// queryContractCapacity query capacity about contract from chain's point of view
+func (uv *UtxoVM) queryContractCapacity(bucket string) (int64, error) {
```

Comment:
It whould be better to move this function to xmodel, and change a name, e.g. GetBucketSize.